modules/ROOT/pages/queries/select-version.adoc

```diff
@@ -115,13 +115,13 @@ For example, if a database is created with Cypher 5 as its default language, pre
 === Procedures and functions
 
 link:{neo4j-docs-base-uri}/operations-manual/current/procedures[Procedures] and xref:functions/index.adoc[functions] (including built-in and link:{neo4j-docs-base-uri}/apoc/current/[APOC]) are tied to a specific Cypher language version.
-Therefore, procedures and functions in Neo4j 2025.04+ and APOC 2025.04+ (both of which have Cypher 25 as their default language) may behave differently depending on what version of Cypher is used.
+Therefore, procedures and functions in Neo4j 2025.06+ and APOC 2025.06+ (both of which have Cypher 25 as their default language) may behave differently depending on what version of Cypher is used.
```

Comment:
will they have Cypher 25 as default? 

I don't know, but if we say databases don't I feel it's slightly odd to say procedures and functions do. Would need looking into

Reply:
I will change - Im pretty sure they will have same as for databases but will doublecheck